worker/draft.go

```diff
@@ -1813,7 +1813,7 @@ func (n *node) InitAndStartNode() {
 
 			// TODO: Making connections here seems unnecessary, evaluate.
 			members := groups().members(n.gid)
-			for _, id := range sp.Metadata.ConfState.Nodes {
+			for _, id := range sp.Metadata.ConfState.Voters {
```

Comment:
@mangalaman93 apologies for not highlighting this earlier, if we could verify that this change in API is legit in a sense that Node -> Voters in v3, then I'd feel more comfortable approving this. Do we have a changelog which points to this?

Reply:
it comes from this PR https://github.com/etcd-io/etcd/pull/10914. Mentioned in the changelog too https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md

Reply:
Exact changelog for the above: https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md#package-raft